bin/dev-cli/src/bridge_out/withdrawal.rs

```diff
@@ -16,7 +16,6 @@ pub(crate) async fn create_withdrawal_transaction(
 ) -> Result<()> {
     // Send the transaction and listen for the transaction to be included.
     let provider = ProviderBuilder::new()
-        .with_recommended_fillers()
```

Comment:
Why was this removed?

Reply:
We had to update `alloy` we did something similar in one of the `reth` updates (which `alloy` was also bumped) in the `strata` codebase.
With the new `alloy` this provider construction was changed.
We did something similar in `faucet-api` also.

In 35a68606f2125c5f8c94f999b576c22a9c5938a2 we bumped `alloy` from `0.3.6` to `0.14.0` (big bump)